providers/nodeReleasesDataProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import { createContext } from 'react';
+import type { FC, PropsWithChildren } from 'react';
+import type {
+  NodeReleaseData,
+  NodeReleasesDataContext as NodeReleasesDataContextType,
+} from '../types';
+
+type NodeReleasesDataProviderProps = PropsWithChildren<{
+  nodeReleasesData: NodeReleaseData[];
+}>;
+
+export const NodeReleasesDataContext =
+  createContext<NodeReleasesDataContextType>(undefined as any);
+
+export const NodeReleasesDataProvider: FC<NodeReleasesDataProviderProps> = ({
+  children,
+  nodeReleasesData,
+}) => {
+  const current = nodeReleasesData.find(
```

Comment:
`nodeReleasesData` should not be a prop, because the Provider itself can fetch the data.

The reason why on the LocaleProvider the i18ndata is a prop is just because the data comes from the pageProps

Reply:
Ah, my reason was for mocking tests, similar to [`EditLink`](https://github.com/nodejs/nodejs.org/blob/main/components/Article/EditLink/__tests__/index.test.tsx#L39-L41).

Reply:
You can easily mock the new Provider you created by mocking the provider itself. (`jest.mock`).